src/http-server/config.js

```diff
@@ -3,9 +3,11 @@ import { appCache } from "../cache.js";
 import { logger } from "../logger.js";
 
 // CONSTANTS
+const kExperimentalWarnings = ["obfuscated-code"];
```

Comment:
You probably need to filter warnings directly from JS-X-Ray warnings: https://github.com/NodeSecure/js-x-ray/blob/master/src/warnings.js#L42

So if tomorrow we add any new experimental warnings no need to be update the code again